config/i18n-tasks.yml

```diff
@@ -98,6 +98,7 @@ ignore_missing:
   - decidim.proposals.application_helper.activity_filter_values.*
   - decidim.content_blocks.*_settings_form.show.*
   - decidim.newsletter_templates.*_settings_form.show.*
+  - actions.invalid_destroy.proposals_count
```

Comment:
Hi @jesusdb! Should this key be preppended with `decidim.meetings.` as in the `ignore_used:` below? or maybe I'm missing something 😄

Reply:
Hi @agustibr!

I prepended it with `decidim.meetings.` as you said, but it was not catching the locale route, so that's why we placed it from `actions.`